benchmarks/torchbench/voirfile.py

```diff
@@ -1,11 +1,28 @@
 def instrument_probe(ov):
-    # ov.probe(
-    #     "/torchbenchmark.util.framework.vision.model_factory/TorchVisionModel/train(data as batch) > #endloop_data as step"
-    # ).give()
-
+    ov.probe("/run/run_one_step > gpu_time").give("gpu_time")
+    ov.probe("/run/run_one_step > cpu_dispatch_time").give("cpu_dispatch_time")
+    ov.probe("/run/run_one_step > cpu_walltime").give("cpu_walltime")
+    ov.probe("/run/run_one_step > tflops").give("tflops")
     (
         ov.probe("/torchbenchmark.util.framework.huggingface.model_factory/HuggingFaceModel/train(self) > #endloop__ as step")
         .augment(batch=lambda self: self.example_inputs["input_ids"])
         .augment(batch_size=lambda batch: len(batch))
         .give()
     )
+    (
+        ov.probe("/torchbenchmark.util.framework.huggingface.model_factory/HuggingFaceModel/train > loss")["loss"]
+        .map(float)
+        .give("loss")
+    )
+    (
+        ov.probe("/torchbenchmark.util.framework.timm.model_factory/TimmModel/train(self) > #endloop__ as step")
+        .augment(batch=lambda self: self.example_inputs)
+        .augment(batch_size=lambda batch: len(batch))
+        .give()
+    )
```

Comment:
The loss is computed in https://github.com/pytorch/benchmark/blob/ff7114655294aa3ba57127a260dbd1ef5190f610/torchbenchmark/util/framework/timm/model_factory.py#L55-L62

It might not be necessary anyway to fetch it as nothing useful is being computed but I'm curious to know how would it be possible to give it

Reply:
I think this might work:

```python
(
    ov.probe("/torchbenchmark/util/framework/timm/model_factory/TimmModel/_step_train(self) > #exit")
    .kmap(loss=lambda self: self.cfg.loss)
    .give()
)
```

Reply:
It gives me a similar error as alternatives I've tried:
```TypeError: float() argument must be a string or a number, not 'CrossEntropyLoss'```

I'm not sure how to specify that I want the result of `self.cfg.loss()` on which `.backward()` is called

Reply:
Oh yeah that's a bit tricky isn't it. `probe("_step_train > CrossEntropyLoss.__call__() as loss")` would work. Or more generically, I think this'll work:

```python
@ov.probe("/.../TimmModel/__init__(self) > #exit").ksubscribe
def reg(self):
    ov.probe("self.cfg.loss.__call__() as loss")
```

Reply:
Thanks it worked! I tried to do the same with `TorchVisionModel`:
```
    @ov.probe("/torchbenchmark.util.framework.vision.model_factory/TorchVisionModel/__init__(self) > #exit").ksubscribe
    def reg_vision_loss(self):
        (
            ov.probe("self.loss_fn.__call__() as loss")
            .take_last(1)["loss"]
            .map(float)
            .give("loss")
        )
```
But no `loss` is feeded (empty sequence). Any ideas?

Reply:
Maybe `self.loss_fn()` would work in this case?

Reply:
Unfortunately not :(
`TypeError: CrossEntropyLoss() cannot be tooled`

Reply:
Got it working with:
```python
    # loss of TorchVisionModel
    @ov.probe("//run_one_step > func").kmap(model=lambda func: func.__self__).ksubscribe
    def reg_vision_loss(model):
        if hasattr(model, "loss_fn"):
            (
                ov.probe("model.loss_fn.__call__() as loss")
                .last()["loss"]
                .map(float)
                .give("loss")
            )
```
but it in the `ov.phases.load_script` phase

Reply:
I guess that's a roundabout way to do it 😂

Reply:
I think it wasn't working in the initial phase because `loss_fn` is not always defined (it gets defined in an `if`)? I was getting some attributes undefined for Model (or similar error)